chapter06/spg/stored_data.json

```diff
@@ -0,0 +1 @@
+{"gilligan.crew.hut":{"ginger.girl.hut":3038169,"gilligan.crew.hut":2801571,"fileserver.copyroom.hut":3019197,"laser3.copyroom.hut":3083400,"professor.hut":3040866,"skipper.crew.hut":2899980,"maryann.girl.hut":3829731},"ginger.girl.hut":{"maryann.girl.hut":3055644,"laser3.copyroom.hut":3234450,"professor.hut":2744034,"skipper.crew.hut":3359298,"ginger.girl.hut":3456060,"gilligan.crew.hut":2979723,"fileserver.copyroom.hut":2847663},"fileserver.copyroom.hut":{"maryann.girl.hut":2969607,"skipper.crew.hut":2868474,"professor.hut":3007113,"laser3.copyroom.hut":3277824,"fileserver.copyroom.hut":3078090,"gilligan.crew.hut":3454062,"ginger.girl.hut":2869914},"professor.hut":{"fileserver.copyroom.hut":2792742,"ginger.girl.hut":3141750,"gilligan.crew.hut":3220671,"skipper.crew.hut":3227079,"professor.hut":3342522,"laser3.copyroom.hut":3054885,"maryann.girl.hut":3368841},"laser3.copyroom.hut":{"maryann.girl.hut":2769888,"ginger.girl.hut":2856003,"gilligan.crew.hut":3089379,"fileserver.copyroom.hut":3021102,"laser3.copyroom.hut":3134451,"professor.hut":2863449,"skipper.crew.hut":3111273},"skipper.crew.hut":{"maryann.girl.hut":2954073,"fileserver.copyroom.hut":3031458,"ginger.girl.hut":3124809,"gilligan.crew.hut":3293646,"skipper.crew.hut":2892486,"laser3.copyroom.hut":3152388,"professor.hut":2863335},"maryann.girl.hut":{"gilligan.crew.hut":3366147,"ginger.girl.hut":3196740,"fileserver.copyroom.hut":3421599,"laser3.copyroom.hut":2958687,"professor.hut":2863122,"skipper.crew.hut":3005610,"maryann.girl.hut":2895729}}
```

Comment:
This works, but it's not very pretty. JSON has parameters to make the output nice and readable if you're into reading JSON files.